src/utils/weatherDataFormatUtils.ts

```diff
@@ -47,21 +47,22 @@ export const weatherConditionsText = (text: string): string => {
     return literals[text];
 };
 
+const options: Intl.DateTimeFormatOptions = {
+    weekday: "short",
+    month: "short",
+    day: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    hour12: false,
+};
+
 export const formatDate = (inputDate: Date): string => {
```

Comment:
Maybe we could rename this to something less generic and more specific since we have moved the options object/enum out of it. What do you think? Maybe in the future there will be a case where we want to have a different date filter or something.

Reply:
oh that's a good idea, do you have any preference? 
no worries if not, I'll have a think

Reply:
Hmm, maybe something like formatToLocaleDate?

Reply:
hmm good suggestion, I'm thinking since we're converting using the US locale, it might be best to mention this in the name. I've pushed a change, let me know your thoughts please

Reply:
I've renamed to `formatDateToUSLocale` for the reason metnioned above, what does that sound to you?

Reply:
It's perfect!